guides/common/modules/con_managing-container-images.adoc

```diff
@@ -4,5 +4,5 @@
 With {Project}, you can import container images from various sources and distribute them to external containers using Content Views.
 
 ifndef::orcharhino[]
-For information about containers, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux_atomic_host/7/html/getting_started_with_containers/index[Getting Started with Containers] in _Red Hat Enterprise Linux Atomic Host 7_.
+For information about containers, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux_atomic_host/7/html/getting_started_with_containers/index[Getting Started with Containers] in _{RHEL} Atomic Host 7_.
```

Comment:
I see it as just a short hand for the full name (not sure why we define the attribute multiple times). IMHO we should later introduce a macro `{EL}` which is `Enterprise Linux` in upstream and `{RHEL}` in Satellite. So I'm not sure I agree with your suggestion here.

And I think the links to access.redhat.com sometimes do make sense, sometimes they don't. In case they're the official documentation about a product and the guide is about syncing that product, it does make sense.

Reply:
OK, then please ignore all my comments with `{RHEL}` in links.

Reply:
I have closed all comments about `{RHEL}` in links.